tests/model/operation/transform/attribute.js

```diff
@@ -376,6 +532,21 @@ describe( 'transform', () => {
 				);
 			} );
 
+			it( 'text into div in different path', () => {
```

Comment:
I don't like this example because it is rather unrealistic. We do not wrap text. And it is bad to have `<p><div></div></p>` structure, casue `<div>` should never be in `<p>`.

Unless it won't have a negative impact on CC, I'd drop this one.

Reply:
I agree with you. I added this kind of tests to treat them as theoretical cases where the range is inside an element. I believe it won't have any impact on CC, so if you're up to remove those (there are more in other files), I'm ok with that.